plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -1035,4 +1408,19 @@ private List<String> getTableFiles(String tableName)
                 .map(path -> format("s3://%s/%s", bucketName, path))
                 .collect(toImmutableList());
     }
+
+    private void assertTableChangesQuery(@Language("SQL") String sql, @Language("SQL") String expectedResult)
+    {
+        assertThat(query(sql))
+                .exceptColumns("_commit_timestamp")
```

Comment:
Is there a test verifying the value in _commit_timestamp makes sense?

for example you could do this by

```sql
  SELECT er.*, th.timestamp AS _commit_timestamp
  FROM (expectedResult) AS er JOIN table$history AS th ON er._commit_id = th.version 
```